README.md

```diff
@@ -23,9 +23,7 @@ If you're submitting a patch which adds/updates npm modules (nodejs- packages) y
 To build locally or release RPMs from this repo, you also require:
 
 * [obal](https://github.com/theforeman/obal) 0.0.2 or higher
```

Comment:
You need an obal release with the un-tito-fication in, which doesn't exist yet ;)